keps/sig-architecture/2140-receipts/README.md

```diff
@@ -0,0 +1,258 @@
+
+To streamline this process and increase transparency, we will add a new Git-based artifact, hereafter referred to as a receipt. This receipt represents an opt-in promise/soft commitment from a SIG about KEPs they're submitting to a specific release and will allow all tracking to occur via Git commits rather than via issue comments, PR references, and other non-auditable mechanisms.
+
+### Goals
+
+* Support better KEP validation and automation through Git-based process
+* Reduce friction and toil early in a release cycle
+* Eliminate instances where Enhancements aren't tracked due to miscommunication
+* Encourage SIGs to update KEPs proactively and plan for release cycles
+* Improve the Exception process turning exception requests into a PR by the SIG with proper approvals
+* Increase KEP / Release transparency
+* Encourage better KEP implementation history updates
+
+
+### Non-Goals
+
+* Generation of receipts for release cycles already closed (Eg. v1.20 and below)
+  * Other than tooling development purposes
+* Generating receipts by RT Enhancements Team on behalf of SIGs and KEP Authors
+
+
+## Proposal
+
+The opting-in process to a release will accept a specified file format (hereafter referred to as a `receipt`) containing metadata about the KEP being enrolled. There would be one receipt per KEP per release cycle.
+
+The receipts would need to be created in the [k/enhancements][k-enhancements] repository. Once a KEP Author/Assignee creates a receipt on behalf of a SIG, it is understood that the feature has been soft opted-in to the release. The opt-in would be finalized only after the Enhancements Team verifies the laid-out acceptance criteria.
+
+The Enhancements Team will continuously monitor the filed receipts during the Enhancements Collection period to verify the graduation requirements. The Enhancements Team will have the power to remove enhancements from the release if they feel they are not conforming to requirements. The team can ask the KEP Assignee to file an exception if the Enhancements Freeze deadline has been passed.
+
+After the collection of Enhancements is completed, KEP Assignees will need to update their receipts with the implementation PRs and documentation PRs (if any). KEP Assignees can update the receipt with the relevant PRs before Enhancements Freeze as well. The Enhancements Team will monitor the mentioned PRs for Code Freeze completion requirements. The team will have the right to ask the assignee to file for an exception or counsel the assignee whether they want to bump the graduation of an enhancement to the next release cycle.
+
+The RT Docs team will monitor the mentioned documentation PRs to see if it conforms to that release cycle's Docs deadline requirements.
+
+[See below][#design-details] for the receipt format and the process.
+
+### User Stories
+
+<!--
+DOUBT: Should we rewrite these into user story format? i.e., "As a SIG tech lead, I want to graduate a KEP in a specific release."
```

Comment:
I think it would _probably_ be useful to list them out like that, but I just like them :) 

I think for these it would probably be useful in the context of the kep to show some of these as workflows or at least expand some and discuss the steps required.